dotcom-rendering/src/web/components/ArticleHeadline.stories.tsx

```diff
@@ -97,7 +97,6 @@ export const ShowcaseInterview = () => (
 							theme: Pillar.Culture,
 						}}
 						tags={[]}
-						isShowcase={true}
```

Comment:
Just wondering why this is getting removed?

Reply:
While the data is passed into the component, the prop isn't actually consumed anywhere within it, meaning it's essentially redundant, causing ESLint to throw an error

Reply:
👍🏻